src/test/java/pl/wrapper/parking/infrastructure/inMemory/ParkingEndpointRepositoryTest.java

```diff
@@ -0,0 +1,71 @@
+package pl.wrapper.parking.infrastructure.inMemory;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+import java.io.File;
+import java.lang.reflect.Field;
```

Comment:
Do not use Reflect. It's never a good idea and it disrupts the test flow. Why are we even using it, by the way? There is no need to.